utils/network.py

```diff
@@ -225,5 +303,6 @@ def construct_network_graph(
         [inds_df, orgs_df, transactions_df]
     )
     G = create_network_graph(aggreg_df)
+    network_metrics(G)
     plot_network_graph(G)
     nx.write_adjlist(G, "Network Graph Node Data")
```

Comment:
This seems unnecessary to write out

Reply:
this was specifically on the nx.write)adjlist(G, "Network Graph Node Data") line

Reply:
oh, that i'm not sure about that then it's alan's code. i can ask him